Mage/src/main/java/mage/abilities/keyword/SpreeAbility.java

```diff
@@ -0,0 +1,28 @@
+package mage.abilities.keyword;
+
+import mage.abilities.StaticAbility;
+import mage.cards.Card;
+import mage.constants.Zone;
+
+/**
+ * @author TheElk801
+ */
+public class SpreeAbility extends StaticAbility {
+
+    public SpreeAbility(Card card) {
+        super(Zone.ALL, null);
+        this.setRuleVisible(false);
+        card.getSpellAbility().getModes().setChooseText("Spree <i>(Choose one or more additional costs.)</i>");
+        card.getSpellAbility().getModes().setMinModes(0);
```

Comment:
My current understanding of Spree is that at least one mode must be chosen for the spell to be cast. So shouldn't that be `setMinModes(1)` here?

Reply:
yeah you're exactly right good catch